### ISSUE TITLE
Fails to bring into canoical form

### ISSUE THREAD

**Issue by stefnotch** [cortex-js/compute-engine#10]
The following test cases fail to be turned into a canoical form

- [ ]  `\displaystyle \left(\sin^{-1}\mleft(x\mright)\right)^{\prime}`
- [ ] `\begin{equation*} 1^{\sin(x)} \end{equation*}`

Tested with https://cortexjs.io/compute-engine/demo/

**Comment by stefnotch** [comment]
Ah, nevermind, they do work. I tested them out on the [compute engine demo page](https://cortexjs.io/compute-engine/demo/) until I realized that the compute engine first tries to bring them into a canonical form

**Comment by arnog** [comment]
Great examples! Thank you.

For some of them, I'm going to have to think a bit about what the proper MathJSON should be, unless you have some suggestions (`f:[a,b]\to\R` for example... would that be considered a function signature?)

**Comment by stefnotch** [comment]
@arnog Yes, I think that `f:[a,b]\to\R` was a function signature. A function `f` that maps from a closed interval `[a,b]` to the real numbers
